docs/reference/sql/functions/date-time.asciidoc

```diff
@@ -22,9 +22,9 @@ The table below shows the mapping between {es} and {es-sql}:
 |==========================
 s|{es}
 s|{es-sql}
-2+h| Index/Table date math
+2+h| Index/Table date/time math
```

Comment:
I would use `datetime` in one word - `/` suggest one can pick either `date` or `time` and the latter it's not actually available.